docs/feature_flag_configuration.md

```diff
@@ -6,24 +6,51 @@ The `FeatureFlagConfiguration` version `v1alpha2` CRD defines the a CR with the
 apiVersion: core.openfeature.dev/v1alpha2
 kind: FeatureFlagConfiguration
 metadata:
-    name: featureflagconfiguration-sample
+  name: featureflagconfiguration-sample
 spec:
-    flagDSpec:
-        envs:
-        - name: FLAGD_PORT
-          value: "8080"
-    featureFlagSpec: 
-        flags: 
-            foo: 
-                state: "ENABLED"
-                variants: 
-                    bar: "BAR"
-                    baz: "BAZ"
-                defaultVariant: "bar"
+  flagDSpec:
+    envs:
+      - name: FLAGD_PORT
+        value: "8080"
+  featureFlagSpec:
+    flags:
+      foo:
+        state: "ENABLED"
+        variants:
+          bar: "BAR"
+          baz: "BAZ"
+        defaultVariant: "bar"
 ```
 
-Within the CRD there are 2 main objects, namely the `flagDSpec` and the `featureFlagSpec`, both offering a different set of configuration for the injected `flagd` sidecars.
+Within the CRD there are 3 main objects, namely the `flagDSpec`, the `featureFlagSpec`, and the `syncProvider`, each offering a different set of configurations for the injected `flagd` sidecars.
```

Comment:
Update this section is you end up remove `flagDSpec` from the example above.

Reply:
ok

Reply:
ok